### DISCUSSION TITLE
AppInsights - logi w różnych regionach

### DISCUSSION THREAD

**Discussion by accent** [AzureCommunityPL/group#108] [Pomoc] [answered]
Mam następujący setup:
2 appki postawione w 2 różnych regionach, każda appka pisze logi do App Insights w swoim regionie. 

Appka ma mało użytkowników, więc te logi nie będą szły w TB czy GB, raczej dziesiątki MB (dziennie).

Chciałbym zbiorczo przez API pobrać logi i wyświetlić w web appce. Jak najlepiej połączyć te 2 źródła?

**Comment by wi5nia** [comment, marked as answer]
To zależy, albo eksportujesz logi do storage I pozniej sobie z nimi robisz co chcesz albo korzystasz z wbudowanych [mechanizmów]( https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/cross-workspace-query)

**Reply by accent** [reply]
dziękuję, o to chodziło, jak szukałem to umknęło mi, że można cross subscription i cross workspace
